CRM/Contact/DAO/ContactType.php

```diff
@@ -308,7 +314,7 @@ public static function &fields() {
         'is_reserved' => [
           'name' => 'is_reserved',
           'type' => CRM_Utils_Type::T_BOOLEAN,
-          'title' => ts('Contact Type is Reserved?'),
```

Comment:
On the others I changed the `<html><label>` but not the title - cos the title shows in (e.g) views where there are multiple entities

Reply:
Ok, but I've taken out the question marks because those are silly

Reply:
Agree